Exiled.Events/Handlers/Player.cs

```diff
@@ -902,5 +908,11 @@ public static class Player
         /// </summary>
         /// <param name="ev">The <see cref="InteractingScp330EventArgs"/> instance.</param>
         public static void OnInteractingScp330(InteractingScp330EventArgs ev) => InteractingScp330.InvokeSafely(ev);
+
+        /// <summary>
+        /// Called before a <see cref="Exiled.API.Features.Player"/> interacts with an Pickup.
```

Comment:
Same as above

Reply:
/// Called before a <see cref="Exiled.API.Features.Player"/> search an Pickup.
is it good ?